components/core/DownloadManager.py

```diff
@@ -7,367 +7,437 @@
 verbose = False
 
 if len(sys.argv) == 2 and sys.argv[1] == '-v':
-    verbose = True
+	verbose = True
 
 threadpool = pool.QueuePool(get_db_con, max_overflow=10, pool_size=20)
 
+
 def add_download(download):
-    db = threadpool.connect()
-    if db is not None:
-        cursor = db.cursor()
-        if download.link.split(':')[0]=="magnet":
-            download_name = download.link.split('&')[1].split('=')[1]
-        else:
-            download_name = download.link.split('/')[-1]
-        sql = "INSERT into download(link, user_name, added_time, download_name) VALUES(%s, %s, %s, %s);"
-        try:
-            cursor.execute(sql, (download.link, download.userName, int(time.time()), download_name))
-            db.commit()
-        except MySQLdb.Error as e:
-            db.rollback()
-            # Shows error thrown up by database
-            print(e)
-            return e[1]
-        return "success"
-    return "db connection error"
+	db = threadpool.connect()
+	if db is not None:
+		cursor = db.cursor()
+		if download.link.split(':')[0] == "magnet":
+			download_name = download.link.split('&')[1].split('=')[1]
+		else:
+			download_name = download.link.split('/')[-1]
+		sql = "INSERT into download(link, user_name, added_time, download_name) VALUES(%s, %s, %s, %s);"
+		try:
+			cursor.execute(sql, (download.link, download.userName, int(time.time()), download_name))
+			db.commit()
+		except MySQLdb.Error as e:
+			db.rollback()
+			# Shows error thrown up by database
+			print(e)
+			return e[1]
+		return "success"
+	return "db connection error"
 
```

Comment:
It shows things right on editing the file with indentation 4, here is show 8 where my original files consider it as 4
@agentmilindu Please suggest something